docs/pages/machine-id/guides/github-actions-kubernetes.mdx

```diff
@@ -0,0 +1,221 @@
+---
+title: Using Machine ID With GitHub Actions and Kubernetes
+description: A tutorial for using Machine ID with GitHub Actions and Kubernetes
+---
+
+<Details
+  title="Version warning"
+  opened={true}
+  scope={["oss", "enterprise"]}
+  scopeOnly={true}
+  min="11.0"
+>
+  Machine ID for GitHub Actions is available starting from Teleport `v11.0`.
+</Details>
+
+GitHub Actions is a popular CI/CD platform that works as a part of the larger
+GitHub ecosystem.
+
+In this guide, we will use Teleport Machine ID and Kubernetes Access to allow a
+GitHub Actions workflow to securely connect to a Kubernetes cluster without the
+need for long-lived secrets.
+
+Teleport supports secure joining on both GitHub-hosted and self-hosted GitHub
+Actions runners as well as GitHub Enterprise Server.
+
+## Prerequisites
+
+(!docs/pages/includes/edition-prereqs-tabs.mdx!)
+
+- A Kubernetes cluster connected to your Teleport cluster. If you do not already
+have one configured, try our
+[Kubernetes Access getting started guide.](../../kubernetes-access/getting-started.mdx)
+In our examples, this Kubernetes cluster will be named `my-kubernetes-cluster`.
+- A local workstation with `tsh` and `tctl` authorized against your Teleport
+cluster as a user with the privileges to create configuration resources.
+- A GitHub repository with GitHub Actions enabled. This guide uses the example
+`gravitational/example`, and this value should be replaced with your own unique
```

Comment:
FWIW, Anyone who tries to look for that repo will be redirected to an actual repo, gravitational/hello. Some users may think that this is a demo repo designed to be used with this guide. I quick look at the lack of recent history *should* dissuade otherwise, but you could avoid this altogether by making this `company/example` or something like that.

Reply:
Hmm, I'm a little worried of using `company/example` since that could genuinely belong to somebody other than us.

Reply:
That's technically possible, but improbable. This is just a thought, do as you like.